app/src/main/java/jwtc/android/chess/ChessImageView.java

```diff
@@ -139,14 +141,11 @@ public void onDraw(Canvas canvas) {
 
 	        bmp = _arrPieceBitmaps[ico._color][ico._piece];
 
-			if(_options._radioBlack != null) {
-				// using if statements to pinpoint if there is another null object.
-				// todo if it's fine then will put back && statements
-				if (_start.sActivity.equals(getContext().getString(R.string.start_play))){
-					if(_options.is_bFlipBlack()){
-						if((_options._radioBlack.isChecked() ? ico._color == 1 : ico._color == 0)) {   // flips black for human vs human without
-							canvas.rotate(180, getWidth() / 2, getHeight() / 2);                                    // autoflip on while in Play mode
-						}
+			// todo if it's fine then will put back && statements
+			if (_start.sActivity.equals(getContext().getString(R.string.start_play))){

```

Comment:
I believe this is still a problem when the start activity is killed by system when resources are low. It should be solved by setting a static property on the ChessImageView by activity main.